### ISSUE TITLE
Query: "RPluMA.R" is missing

### ISSUE THREAD

**Issue by MSJennyLee** [movingpictures83/PhyloSeq2LEfSe#1]
is:issue is:open Question: It seems that "RPluMA.R" is not included. How can I run the code "source("RPluMA.R")" in “PhyloSeq2LEfSePlugin.R"?

**Comment by movingpictures83** [comment]
Are you trying to run this standalone?  This code is designed to be a plugin for PluMA (http://biorg.cs.fiu.edu/pluma).  If you set up a pipeline with PluMA and include this plugin, RPluMA.R will be present.  The example/ folder includes a PluMA-compatible input configuration file.

Standalone execution may not be impossible (never have tested it), but you'd need to remove the reference to RPluMA.R and all references to pfix, which calls prefix() from the RPluMA library.  This prepends an input and output file prefix used by PluMA when running pipelines.

**Comment by MSJennyLee** [comment]
> Are you trying to run this standalone? This code is designed to be a plugin for PluMA (http://biorg.cs.fiu.edu/pluma). If you set up a pipeline with PluMA and include this plugin, RPluMA.R will be present. The example/ folder includes a PluMA-compatible input configuration file.
> 
> Standalone execution may not be impossible (never have tested it), but you'd need to remove the reference to RPluMA.R and all references to pfix, which calls prefix() from the RPluMA library. This prepends an input and output file prefix used by PluMA when running pipelines.

Yes, I tried to run “PhyloSeq2LEfSePlugin.R" standalone. Then according to your comment, I downloaded "PluMA-Windows.zip" from "http://biorg.cs.fiu.edu/pluma", unzipped it, and put the file folder "PhyloSeq2LEfSe-main" (unzipped) in the plugins/ directory of the PluMA (C:\Users\LSZ\OneDrive - csu.edu.cn\Desktop\PluMA\plugins\PhyloSeq2LEfSe-main). Then I double-clicked "PhyloSeq2LEfSePlugin.R" and opened Rstudio (R version 4.1.1, 64 bit. Rstudio version 1.4.1717). When it ran to the code "dyn.load(paste("RPluMA", .Platform$dynlib.ext, sep=""))", error appeared (Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/LSZ/OneDrive - csu.edu.cn/Desktop/PluMA/plugins/PhyloSeq2LEfSe-main/RPluMA.dll':
  LoadLibrary failure: The specified module could not be found.)
![微信图片_20210923110717](https://user-images.githubusercontent.com/26650947/134449484-5b2b9d59-58fb-4fd9-b375-60f8d5633873.png)

There was no "RPluMA.dll" in the downloaded file folder PluMA-Windows.zip.
Do I need to install PluMA first? How? I double-clicked "pluma.exe" and "PluMa Windows.exe" but there were no responses.

**Comment by movingpictures83** [comment]
PluMA-Windows needs to be run through 'cmd' at the moment.  You would type "pluma.exe" followed by its input configuration file (which is where you specify the PhyloSeq2LEfSe plugin, see the PluMA documentation).

We intend to improve this interface in the future; PluMA-Windows is a preliminary alpha release that was just released this month.
